src/main/java/frc/robot/subsystems/LED.java

```diff
@@ -0,0 +1,88 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.constants.Constants;
+import frc.robot.constants.IdConstants;
+
+import com.ctre.phoenix.led.CANdle;
+import com.ctre.phoenix.led.CANdle.LEDStripType;
+import com.ctre.phoenix.led.CANdle.VBatOutputMode;
+import com.ctre.phoenix.led.ColorFlowAnimation.Direction;
+import com.ctre.phoenix.led.CANdleConfiguration;
+import com.ctre.phoenix.led.ColorFlowAnimation;
+import com.ctre.phoenix.led.FireAnimation;
+import com.ctre.phoenix.led.LarsonAnimation;
+import com.ctre.phoenix.led.LarsonAnimation.BounceMode;
+import com.ctre.phoenix.led.RainbowAnimation;
+import com.ctre.phoenix.led.RgbFadeAnimation;
+import com.ctre.phoenix.led.SingleFadeAnimation;
+import com.ctre.phoenix.led.StrobeAnimation;
+import com.ctre.phoenix.led.TwinkleAnimation;
+import com.ctre.phoenix.led.TwinkleAnimation.TwinklePercent;
+import com.ctre.phoenix.led.TwinkleOffAnimation;
+import com.ctre.phoenix.led.TwinkleOffAnimation.TwinkleOffPercent;
+
+public class LED extends SubsystemBase {
+
+    private CANdle candle;
+    private static final int stripLength = 67;
+
+    // private CANdleConfiguration config;
+
+    // constructor
+    public LED() {
+        this.candle = new CANdle(IdConstants.CANDLE_ID, "rio");
+        // this.config = new CANdleConfiguration();
+
+        // config.statusLedOffWhenActive = true;
+        // config.disableWhenLOS = false;
+        // config.stripType = LEDStripType.RGB;
+        // config.brightnessScalar = 1;
+        // config.vBatOutputMode = VBatOutputMode.Modulated;
+        // config.v5Enabled = false;
+
+        // candle.configAllSettings(config, 10000);
+
+        candle.configStatusLedState(false);
+        candle.configLOSBehavior(false);
+
+        System.out.println("Strip type ec: " + candle.configLEDType(LEDStripType.GRB));
+
+        candle.configBrightnessScalar(1);
+        candle.configVBatOutput(VBatOutputMode.On);
+        candle.configV5Enabled(true); // Turns off LEDs
+    }
+
+    @Override
+    public void periodic() {
+    }
+
+    // public enum Colors {
+    // RED({255, 0, 0,}),
+    // GREEN({0, 255, 0}),
+    // BLUE({0, 0, 255}),
+    // ORANGE({252, 111, 3}),
+    // OFF({0, 0, 0});
+
+    // private int[] colorValue;
+
+    // private Colors(int[] colorValue) {
+    // this.colorValue = colorValue;
+    // }
+    // }
+
+    public void setSection(int r, int g, int b, int start, int end) {
```

Comment:
Since we aren't using the periodic method anymore, maybe add a method to set the color or animation of all of the LEDs. You can call these setLEDs() and animate(), and they should call the simplest version of the same methods on the candle and have the same parameters.